### PR TITLE
[Feature] Improve AWS node serve dynamic html via http

### PR DESCRIPTION
**Pull request by Pigius**
Hi, @pgrzesik small improvements:

- added SLS framework v.3
- added packages
- improved documentation
- etc

### CODE REVIEW

**Comment by pgrzesik** [comment]
Hey @Pigius - I don't have access to this repo anymore, you might need to request @mnapoli or @medikoo for a review here

**Comment by Pigius** [comment]
@mnapoli WDYT?

**Comment by mnapoli** [comment]
Hi, I'm no longer a maintainer on this repo either, sorry!

File: aws-node-serve-dynamic-html-via-http-endpoint/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,1113 @@
+{
```
Comment (medikoo) [inline aws-node-serve-dynamic-html-via-http-endpoint/package-lock.json:1]: Let's not include `package-lock.json`

File: aws-node-serve-dynamic-html-via-http-endpoint/serverless.yml [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
-service: serve-dynamic-html-via-http-endpoint
+service: serve-dynamic-html-via-http
 
-frameworkVersion: ">=1.1.0 <2.0.0"
+frameworkVersion: "3"
 
 provider:
   name: aws
-  runtime: nodejs12.x
+  runtime: nodejs14.x
```
Comment (medikoo) [inline aws-node-serve-dynamic-html-via-http-endpoint/serverless.yml:8]: Maybe let's use `nodejs16.x`, it'll extend lifespan of this template

**Review by medikoo — COMMENTED** [review]
Thank you @Pigius and sorry for the long wait

File: aws-node-serve-dynamic-html-via-http-endpoint/serverless.yml (hunk 2 above)
Comment (Pigius) [reply aws-node-serve-dynamic-html-via-http-endpoint/serverless.yml:8]: I've tried with runtime: nodejs16.x @medikoo but no lack:

```
Warning: Invalid configuration encountered
  at 'provider.runtime': must be equal to one of the allowed values [dotnetcore3.1, go1.x, java11, java8, java8.al2, nodejs12.x, nodejs14.x, provided, provided.al2, python3.6, python3.7, python3.8, python3.9, ruby2.7]

Learn more about configuration validation here: http://slss.io/configuration-validation
```

File: aws-node-serve-dynamic-html-via-http-endpoint/package-lock.json (hunk 1 above)
Comment (Pigius) [reply aws-node-serve-dynamic-html-via-http-endpoint/package-lock.json:1]: Fixed

File: aws-node-serve-dynamic-html-via-http-endpoint/serverless.yml (hunk 2 above)
Comment (medikoo) [reply aws-node-serve-dynamic-html-via-http-endpoint/serverless.yml:8]: @Pigius Be sure to use latest version of the Framework

Comment (Pigius) [reply aws-node-serve-dynamic-html-via-http-endpoint/serverless.yml]: @medikoo you're right! 😓 fixed!

**Review by medikoo — APPROVED** [review]
Thank you @Pigius 👍

**Pull request merged**